content/transform_keywords_to_grammar.py

```diff
@@ -0,0 +1,69 @@
+from pathlib import Path
```

Comment:
This file is hard to review, just an extract from yaml_transformer.py or some real changes?

Reply:
just a rename and a bit of cleanup (there is an open discussion whether we want to keep this at all, but for now let's leave it: https://github.com/Felienne/hedy/discussions/2621)